docs/framework/additional-apis/_httpresponse.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "_HttpResponse Field | Microsoft Docs"
+ms.custom: ""
+ms.date: "04/24/2017"
+ms.prod: ".net-framework"
+ms.reviewer: ""
+ms.suite: ""
+ms.technology:
+  - "dotnet-winforms"
+ms.tgt_pltfrm: ""
+ms.topic: "article"
+apiname:
+  - "System.Net"
+apilocation:
+  - "System.dll"
+apitype: "Assembly"
+ms.assetid: eab9b789-beb4-4c28-b2d8-78debc7ba129
+caps.latest.revision: 3
```

Comment:
you can remove this metadata and the other empty ones we don't generally use. they come with the migrated topics